src/models/github-server-app.ts

```diff
@@ -48,6 +48,26 @@ export class GitHubServerApp extends Model {
 		});
 	}
 
+	/**
+	 * Get all GitHubServerApps for gitHubBaseUrl
+	 *
+	 * @param {{gitHubBaseUrl: string}} gitHubBaseUrl
+	 * @returns {GitHubServerApp[]}
+	 */
+	static async getAllForGitHubBaseUrl(
```

Comment:
New method to allow us to query the GHSA table to see if there are existing GH apps for the provided URL.